SnapsInAZfs/ConfigConsole/ConfigConsole.cs

```diff
@@ -54,6 +54,7 @@ public static void RunConsoleInterface( IZfsCommandRunner commandRunner )
 
         CommandRunner = commandRunner;
 
+        Application.Init();
```

Comment:
Well that's probably some bug. I'll investigate it.

Reply:
It is not a bug. You really can remove the `Init` call from the project. What is causing the error on the master branch is the `IsMdiContainer = true;`. Removing it doesn't throw the error but also the windows will not work well because of the reasons you already know.

https://github.com/snapsinazfs/snapsinazfs/blob/52ea515181bffd2454ab6bdd478635f0b2ab64c0/SnapsInAZfs/ConfigConsole/SnapsInAZfsConfigConsole.cs#L376

Reply:
I'll be interested to see what happens once the next release of Terminal.Gui with the fixes is out. Should be easier to track down any remaining bad behavior in my code at that point, as well.